Dockerfile-fpm

```diff
@@ -33,11 +33,13 @@ COPY src/php/conf/ /usr/local/etc/php/conf.d/
 COPY src/php/fpm/conf/*.conf /usr/local/etc/php-fpm.d/
 
 # Install shush
-COPY src/php/utils/install-shush /usr/local/bin/
+COPY src/php/utils/install-* /usr/local/bin/
```

Comment:
It's not really part of installing shush anymore, it's now copying over all installs. Do we not want to be explicit about this?

Reply:
wrong move :)